modules/apps/commerce/commerce-product-type-grouped-web/src/main/resources/META-INF/resources/view_cp_definition_grouped_entries.jsp

```diff
@@ -210,39 +214,40 @@ renderResponse.setTitle(cpDefinition.getName(themeDisplay.getLanguageId()));
 	}
 </aui:script>
 
-<aui:script use="liferay-item-selector-dialog">
-	window.document
-		.querySelector('#<portlet:namespace />addDefinitionGroupedEntry')
-		.addEventListener('click', (event) => {
-			event.preventDefault();
+<aui:script sandbox="<%= true %>">
+	const addDefinitionGroupedEntryButton = document.getElementById('<portlet:namespace />addDefinitionGroupedEntry');
 
-			var itemSelectorDialog = new A.LiferayItemSelectorDialog({
-				eventName: 'productDefinitionsSelectItem',
-				on: {
-					selectedItemChange: function (event) {
-						var <portlet:namespace />addCPDefinitionIds = [];
+	if (addDefinitionGroupedEntryButton) {
+		addDefinitionGroupedEntryButton.addEventListener('click', (event) => {
+			event.preventDefault();
 
-						var selectedItems = event.newVal;
+			Liferay.Util.openSelectionModal({
+				multiple: true,
+				onSelect: function (selectedItems) {
+					if (selectedItems?.length) {
+						const entryCPDefinitionIds = document.getElementById(
+							'<portlet:namespace />entryCPDefinitionIds'
+						);
 
-						if (selectedItems) {
-							window.document.querySelector(
-								'#<portlet:namespace />entryCPDefinitionIds'
-							).value = selectedItems;
+						if (entryCPDefinitionIds) {
+							entryCPDefinitionIds.value = selectedItems.map(item => item.value);
+						}
 
-							var addCPDefinitionGroupedEntryFm = window.document.querySelector(
-								'#<portlet:namespace />addCPDefinitionGroupedEntryFm'
-							);
+						const addCPDefinitionGroupedEntryFm = document.getElementById(
+							'<portlet:namespace />addCPDefinitionGroupedEntryFm'
+						);
 
+						if (addCPDefinitionGroupedEntryFm) {
 							submitForm(addCPDefinitionGroupedEntryFm);
 						}
-					},
+					}
 				},
+				selectEventName: 'productDefinitionsSelectItem',
```

Comment:
If this is a standard multiple selection, we can remove `selectEventName`

Reply:
👍